dotlottie-rs/src/dotlottie_player.rs

```diff
@@ -978,6 +1009,14 @@ impl DotLottiePlayer {
             .retain(|o| !Arc::ptr_eq(o, observer));
     }
 
+    pub fn load_theme(&self, theme_id: &str) -> bool {
```

Comment:
Just wondering why we're not utilising `setConfig` for setting themes as well.

Reply:
@afsalz for consistency with other methods (load_animation & load_animation_data) we use to load internal or external animation assets, a theme is just another asset in the .lottie file. Also, `set_config` doesn't provide feedback for the success or failure of the operation.